internal/search/query/predicate_test.go

```diff
@@ -179,8 +179,8 @@ func TestRepoDescriptionPredicate(t *testing.T) {
 		}
 
 		valid := []test{
-			{`literal`, `test`, &RepoDescriptionPredicate{}},
-			{`regexp`, `test(.*)package`, &RepoDescriptionPredicate{}},
+			{`literal`, `test`, &RepoDescriptionPredicate{Pattern: "test"}},
+			{`regexp`, `test(.*)package`, &RepoDescriptionPredicate{Pattern: "test(.*)package"}},
```

Comment:
Could we add a test that exercises the "join with `.*`" behavior?

```suggestion
			{`regexp`, `test(.*)package`, &RepoDescriptionPredicate{Pattern: "test(.*)package"}},
			{`regexp`, `test package`, &RepoDescriptionPredicate{Pattern: "test(?:.*)package"}},
```

Reply:
So that behavior happens [here](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@1af130cf83e7fb82cee71d54dc0d795d4dcabec8/-/blob/internal/search/query/types.go?L396) in `Description()`
Added a test for it [here](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@1af130cf83e7fb82cee71d54dc0d795d4dcabec8/-/blob/internal/search/query/types_test.go?L8)